notebooks/growth_model_generate_data.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<p>maybe its worth mentioning here that you will need your own API key and can be setup by following the instructions here: <a href="https://platform.openai.com/docs/quickstart?context=python" target="_blank">https://platform.openai.com/docs/quickstart?context=python</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Will do. thanks.</p><p> </p>